.idea/externalDependencies.xml

```diff
@@ -2,6 +2,6 @@
 <project version="4">
   <component name="ExternalDependencies">
     <plugin id="com.dubreuia" />
-    <plugin id="google-java-format" />
+    <plugin id="google-java-format" min-version="1.12.0.0"/>
```

Comment:
🔧 Please add a comment specifying the extra .0 is required to prevent confusion

Reply:
I chose not to add the comment, see https://github.com/camunda/zeebe/pull/9032#discussion_r840553278